### ISSUE TITLE
Town of Us Mod Stuttering

### ISSUE THREAD

**Issue by KH0DIIN** [polusgg/Town-Of-Us#127]
@slushiegoose while i'm normally playing the Town of Us mod my game keeps stuttering every 0.5 seconds and when i try playing the normal Among Us my game is on 60 FPS straight doesn't even go to 59 for 0.2 seconds but in the Town of Us my FPS is 50 and goes to 30 and while it's on 50 i get stuttering every single 0.5 seconds in a game with Town of Us my game stutters which made me so mad i literally deleted the mod so please fix the stuttering issue, i'm not the only one who faces this literally it stutters for no damn reason
![image](https://user-images.githubusercontent.com/77162835/116403995-7fbd3380-a82e-11eb-8908-f53e869b20ee.png)
![image](https://user-images.githubusercontent.com/77162835/116404033-89df3200-a82e-11eb-9af5-98417d524842.png)
![image](https://user-images.githubusercontent.com/77162835/116404054-8fd51300-a82e-11eb-98ba-b7f772f14a28.png)
in the pictures i've added you can see i'm not the only one who faces the stutter issue so please fix that soon, thanks

**Comment by MrFawkes1337** [comment]
I lag even with normal Among Us on official servers, using my personal Impostor server everything is fine even with mod, so maybe try a custom server, and see if you still crash.

**Comment by KH0DIIN** [comment]
i try doing everything, i've tried creating lobbies and i've tried turning on VSync but when i tried changing system graphics it doesn't stutter anymore it only does when a camouflage happens or when comms make you camouflaged that's when my FPS goes to 30 and 40 but normally now my FPS is 60 all fine with my normal graphics, thanks anyways

**Comment by cheickit** [comment]
I'll add my own testimony to this : i don't think this is specs related, task manager is fine on my end, i tried a fresh install but no luck. Changing nvidia settings make it worse for me personally.

**Comment by FATE-Punky** [comment]
I am also having this problem. I have tried fresh reinstallation and whatnot and nothing has fixed the issue. Two other individuals that I play the game with have similar issues. It doesn't seem like it is tasking on my system or anything. I have no issues with the base game. Let me put it this way, I can play Cyberpunk on here almost flawlessly. After first encountering the issue within the last two weeks, I decided to revisit it after the recent update and it has not improved for me.
